src/Miunie.Discord/CommandHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+
+namespace Miunie.Discord
+{
+    public class CommandHandler
+    {
+        private CommandsNextModule _commandsNextModule;
+
+        public CommandHandler(CommandsNextModule commandsNextModule)
+        {
+            _commandsNextModule = commandsNextModule;
+        }
+
+        public async Task InitializeAsync()
+        {
+            /*
+             TODO (Charly): Here we should register the commands, converters...
+             using the _commandService.RegisterCommands method
+             https://dsharpplus.emzi0767.com/api/DSharpPlus.CommandsNext.CommandsNextModule.html#DSharpPlus_CommandsNext_CommandsNextModule_RegisterCommands_Assembly_
+            */
```

Comment:
This also feels like an empty proxy and I think we should get rid of this class.
The `CommandHandler` was meant as the command handler system that comes with DSharpPlus.

If you say that only `CommandsNextModule` is needed, we should just have that directly in `MiunieDiscord`